### ISSUE TITLE
Prevent multiple reloads

### ISSUE THREAD

**Issue by adamchainz** [adamchainz/django-browser-reload#22]
On my DB Buddy project I'm seeing many reloads for a single file changes. It looks like watchman is signalling the same file changing repeatedly.

Maybe something needs changing in this project, or we need to add debounce logic.

**Comment by adamchainz** [comment]
There’s a 50ms denounce: https://github.com/adamchainz/django-browser-reload/blob/c6811289426bb2a2acc959627766baf733ae828b/src/django_browser_reload/views.py#L46

can you try adjusting that in your project to see if it helps? Maybe we can make it configurable.

**Comment by Mte90** [comment]
As I can see avdon't works, I changed to 10 seconds instead the refresh is like fast.
Maybe because `runserver` refresh for all of them?

**Comment by adamchainz** [comment]
Ah right, if runserver restarts, the debounce does not apply. Can you describe your situation a bit more? Is your runserver restarting multiple times?

**Comment by Mte90** [comment]
Yes runserver restart multiple times because I have pylint/pydocstyle and so on that on every save change the file so runserver is execute multiple times.
Maybe is better a queue in runserver?

**Comment by adamchainz** [comment]
That would need implementing in Django itself.

we could perhaps add a debounce on the JavaScript side for restart events only. Would you like to give that a try?
